frontend/app_flowy/packages/flowy_infra/lib/theme.dart

```diff
@@ -141,9 +141,9 @@ class AppTheme {
           primaryVariant: main2,
           secondary: main2,
           secondaryVariant: main2,
-          background: bg1,
+          background: surface,
           surface: surface,
-          onBackground: bg1,
+          onBackground: surface,
```

Comment:
Make sure these colors you're filled also work on light mode.

Reply:
Was following Bran's Color Scheme in the [figma](https://www.figma.com/file/Ty2OmskN8oKgj2fLrqGer9/Appflowy.io-(Community)?node-id=2%3A307) document. With the naming conventions was not sure which color goes where.